lib/metadata/bucketclient/backend.js

```diff
@@ -1,15 +1,23 @@
 import assert from 'assert';
 
 import bucketclient from 'bucketclient';
+import { errors } from 'arsenal';
 import BucketInfo from '../BucketInfo';
 import { logger } from '../../utilities/logger';
 import config from '../../Config';
 
+const METADATA_DEFAULT_VERSION = 1;
+
 class BucketClientInterface {
     constructor() {
         assert(config.bucketd.bootstrap.length > 0,
                'bucketd bootstrap list is empty');
         const { bootstrap, log } = config.bucketd;
+        this.metadataVersion = METADATA_DEFAULT_VERSION; // 6.3.0 or earlier
+        this.gotMetadataInfo = false; // got metadata information ?
+                                      //     go ahead sending requests : wait
+        // TODO remove when getURIComponents has been fixed
+        this.extendedVersion = false;
```

Comment:
It's a flag saying whether the metadata deployment is supporting versioning or not (now metadata's getURIComponents is incompatible with versioning, see https://github.com/scality/MetaData/pull/706).